frontend/src/components/Upgrade.tsx

```diff
@@ -51,10 +51,7 @@ export default function Upgrade({ up, status, onBuy, currencyAmount }: IUpgradeP
 
   return (
     <div className={`upgrade status-${status}`} onClick={onClickEvent} style={{ background: bgColor }}>
-      <span className="upgrade-image">
-        <UpgradeImage picture={up.picture}></UpgradeImage>
-      </span>{" "}
-      {nameText} {percentageText}
+      <UpgradeImage picture={up.picture}></UpgradeImage> {nameText} {percentageText}
```

Comment:
Is there any difference that the span is removed for string images now? I see the span is brought back for PNGs but for strings it's entirely absent.

Reply:
@jvgusch It was a mistake, should be even better now.

Reply:
Actually nevermind